host/Web.go

```diff
@@ -20,6 +20,9 @@ type Web struct {
 
 // Host blocks processing web requests until it ends, at which point it signals the indicated WaitGroup
 func (web Web) Host(wait *sync.WaitGroup) {
+	fs := http.FileServer(http.Dir("./api/swagger-ui"))
+	http.Handle("/swaggerui/", http.StripPrefix("/swaggerui/", fs))
```

Comment:
wdyt about using `/v1alpha/swaggerui` instead? and perhaps to change Web.go line 26 to also be `/v1alphapermissions/check` instead, so we're consistent?

Reply:
Personal opinion: adding something to the route that says it's alpha makes a ton of sense ... after code generation is in. The main benefit is being able to tinker freely like that. :)